rspirv/dr/constructs.rs

```diff
@@ -392,9 +392,11 @@ mod tests {
         assert_eq!(
             format!(
                 "{}",
-                dr::Operand::FunctionControl(spirv::FunctionControl::INLINE)
+                dr::Operand::FunctionControl(
+                    spirv::FunctionControl::INLINE | spirv::FunctionControl::CONST
+                )
             ),
-            "INLINE",
+            "INLINE | CONST",
```

Comment:
Here for example, isn't it useful to know that we're printing an _`Operand` of variant `FunctionControl`_?

Reply:
Ok, I adjust the testcase to add the struct name and using the default `Debug` impl. (Initially I thought this testcase was for pretty-printing in disassembly but this seems to be separate)

Reply:
Lovely, thanks so much!